ignitionrobotics/ign-gazebo
# Added altimeter sensor inspector

Signed-off-by: Nate Koenig <nate@openrobotics.org>

# 🎉 New feature

## Summary

Builds on #1171 by adding a GUI inspector for altimeters.

This PR also demonstrates how to add a new component inspector element. 

## Test it

1. `ign gazebo -v 4 sensors.sdf`
2. Take note of the output from `ign model -m sensors_box -l link -s altimeter`
3. Select the altimeter sensor in the entity tree.
4. Make some changes to the values in the component inspector. 
5. Rerun `ign model -m sensors_box -l link -s altimeter` and you should see the changes.

## Checklist

**Note to maintainers**: Remember to use **Squash-Merge**
